### DISCUSSION TITLE
Show altitude and external sensor temperature on Apple CarPlay screen

### DISCUSSION THREAD

**Discussion by Stew503** [osmandapp/OsmAnd#18955] [General]
Hello. I love your product. It has helped me navigate thousands of miles of backcountry on my motorcycle. Is there a way to display the altitude on the Apple CarPlay screen? I’d also like to display the temperature from an external sensor on the Apple CarPlay screen as well. Please let me know. Thanks!!!

**Comment by DmitryAlexei** [comment]
As of now, due to Car play limitations, Osmand only displays alert widget information on the car screen (see documentation https://osmand.net/docs/user/navigation/car-play#screen-alert-widget)

You can find all open issues related to Car Play by the link https://github.com/osmandapp/OsmAnd-iOS/issues?q=is%3Aopen+is%3Aissue+label%3Acar-play
